requirements_test.txt

```diff
@@ -0,0 +1,13 @@
+func_timeout
+numpy
+openai
+pandas
+psycopg2-binary
+pysqlite3
+sentence_transformers
+snowflake-connector-python
+spacy==3.7.2
```

Comment:
Ok so after a lot of experimentation it turns out that spacy==3.7 is required for the tests to work as before